docs/user_documentation/ref-emx.md

```diff
@@ -191,7 +191,7 @@ true/false to indicate a readOnly attribute
 ability to tag the data referring to the tags sections, described below
 
 ### validationExpression 
-javascript validation expression that must return a bool. Must return true if valid and false if invalid. See for a syntax description the section [[Javascript Expressions]]
+javascript validation expression that must return a bool. Must return true if valid and false if invalid. See for a syntax description the section [Expressions](ref-expressions)
```

Comment:
See for a syntax description the section [Expressions](ref-expressions) -> See the [Expressions](ref-expressions) section for a syntax description.

Reply:
Agreed. Feel free to make these minor edits yourself.